### ISSUE TITLE
cron with secondary container not working

### ISSUE THREAD

**Issue by noplanman** [friendica/docker#35]
When I try to use a secondary container with `/cron.sh` entrypoint, all I get is:
`Friendica isn't properly installed yet.`

I'm passing the same environment variables as the main docker container, so not sure where the hiccup is happening.

**Comment by nupplaphil** [comment]
Did you use some of the `.examples` as baseline? Can you paste your `docker-compose.yml` if possible or the startup environment variables.

**Comment by Zauberstuhl** [comment]
FYI it worked for me after I added:

```
[system]
pidfile = "/var/run/friendica.pid"
```

to `config/local.ini.php`

**Comment by Zauberstuhl** [comment]
@noplanman the cron.sh starts a daemon script. if you want everything on one container it might make sense to alter the entrypoint and start the workers permanently via:

`php /var/www/html/bin/daemon.php -f start`

Then you have no host dependencies.

**Comment by iannuzzelli** [comment]
I had a very similar issue and ended up creating the cronjob in the host as suggested by @noplanman , although not nice as it creates a dependency in case of future scaling needs.

In my current setup (using `friendica:2020.03-apache` image), the cron container, despite being passed the same env file of the friendica app, doesn't start the daemon, it just hangs reporting this warning

```
[WARN  tini (253)] Tini is not running as PID 1 and isn't registered as a child subreaper.
Zombie processes will not be re-parented to Tini, so zombie reaping won't work.
To fix the problem, use the -s option or set the environment variable TINI_SUBREAPER to register Tini as a child subreaper, or run Tini as PID 1.
```

then it tries to send the error notification via email, using the wrong SMTP (I believe that's because the SMTP env variables are not parsed by cron.sh, which is probably a separate issue)

**Comment by nupplaphil** [comment]
- The Zombie issue is resolved by using `gosu` and `tini`, changed #162 - it never happened again since this change
- The `pidfile` is now part of the core config, so no need to add it extra (https://github.com/friendica/docker/blob/stable/.config/zz-docker.config.php#L13)

So I'm closing this issue :)
